ATTRIBUTION.md

```diff
@@ -17,6 +17,120 @@ This package contains a variety of materials, including data sets and related ma
+* Exact source location: https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html
+#### Destination
+* Modifications to source files(s): Downstate subregions G_I are made by grouping and splitting county shapes.
+* Location: ***prereise/gather/data/remap_ba_area/nyiso/NYISO_downstate.geojson***
+
+
+#### CAISO
+#### Source
+* Name: subregions of CAISO
+* Author: CAISO
+* Description: the subregions of CAISO
+* Source: https://www.caiso.com/
+* Exact source location: https://cecgis-caenergy.opendata.arcgis.com/datasets/CAEnergy::california-electric-balancing-authority-areas
+#### Destination
+* Modifications to source files(s): select county shapes from the image map
+* Location: ***prereise/gather/data/remap_ba_area/CA_balancing_authorities/***
+
+#### PJM
+#### Source
+* Name: subregions of PJM
+* Author: PJM
+* Description: the subregions of PJM
+* Source: PJM
+* Exact source location:  https://www.monitoringanalytics.com/reports/PJM_State_of_the_Market/2018/2018-som-pjm-volume2-appendix.pdf
+#### Destination
+* Modifications to source files(s): The shape files are made from the image.
+* Location: ***prereise/gather/data/remap_ba_area/pjm.zip***
+
+#### ISONE
+#### Source
+* Name: subregions of ISONE
+* Author: ISONE
+* Description: the subregions of ISONE
+* Source: ISONE
+* Exact source location: https://www.iso-ne.com/isoexpress/web/charts
+#### Destination
+* Modifications to source files(s): the state maps are used
+* Location: ***prereise/gather/data/remap_ba_area/isone.zip***
+
+#### ERCOT
+https://www.ercot.com/files/docs/2021/12/23/2021_Report_Existing_Potential_Electric_System_Constraints_Needs.pdf
+#### Source
+* Name: subregions of ERCOT
+* Author: ERCOT
+* Description: the subregions of ERCOT
+* Source: https://www.ercot.com
+* Exact source location: https://www.ercot.com/news/mediakit/maps/index.html
+#### Destination
+* Modifications to source files(s): the county map is picked to form the subregions based on the ERCOT image.
+* Location: ***prereise/gather/data/remap_ba_area/ercot_zone_mapping.csv***
+
+#### SWPP
+#### Source
+* Name: subregions of SWPP
+* Author: SWPP
+* Description: the subregions of SWPP
+* Source: https://spp.org
+* Exact source location: https://spp.org/documents/31587/spp101%20-%20an%20introduction%20to%20spp%20-%20all%20slides%20print.pdf
+#### Destination
+* Modifications to source files(s): the county maps are selected based on the image map
+* Location: ***prereise/gather/data/BA_County_map.json***
+
+#### MISO
+#### Source
+* Name: subregions of MISO
+* Author: MISO
+* Description: the subregions of MISO
+* Source: https://cdn.misoenergy.org
+* Exact source location: https://cdn.misoenergy.org/MISO%20Regional%20and%20Interregional%20Cost%20Allocation%20Reference90295.pdf
+#### Destination
+* Modifications to source files(s): the county maps are selected based on the image map
+* Location: ***prereise/gather/data/BA_County_map.json***
```

Comment:
It seems that we aren't making any changes to the existing sub-region mapping within SPP and MISO that's already part of `BA_County_map.json`, is that correct?

Reply:
Correct.  The MISO and SWPP sections are removed from the attribution.md file.